File: Core/Entities/SBUWiseBudget.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Core.Entities
{
    public class SBUWiseBudget : BaseEntity
    {
        public string SBU { get; set; }
        public string SBUName { get; set; }
        public int DonationId { get; set; }
        [ForeignKey("DonationId")]
        public Donation Donation { get; set; }
        public DateTimeOffset? FromDate { get; set; }
        public DateTimeOffset? ToDate { get; set; }
        public long Amount { get; set; }
        public string Remarks { get; set; }
    }
}

File: client/src/app/shared/models/sbuWiseBudget.ts
import { IDonation } from "./donation";
import { ISBU } from "./sbu";

export interface ISBUWiseBudget {
    id: number;
    sbu: string;
    sbuName: string;
    donationId: number;
    amount: number;
    fromDate: any;
    toDate: any;
    status: string;
    year: any;
    donation:IDonation;
}

export class SBUWiseBudget implements ISBUWiseBudget {
    id: number = 0;
    sbu: string=null;
    sbuName: string;
    donationId: number=null;
    amount: number;
    fromDate: any;
    toDate: any;
    status: string;
    year: any;
    donation:IDonation;
}



